### PR TITLE
register pass compatibility 

### PR DESCRIPTION
**Pull request by cryoco**
### PR types

Others
### PR changes

Others
### Describe

1. register pass compatibility for `conv_bn_fuse_pass`, `conv_affine_channel_fuse_pass`, `repeated_fc_relu_fuse_pass`, `shuffle_channel_detect_pass`
2. fix bugs of `repeated_fc_relu_fuse_pass` when input dims is not 2

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_affine_channel_fuse_pass.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+        self.feeds = {
+            "data": np.random.random([1, 3, 64, 64]).astype("float32"),
+        }
+        self.fetch_list = [ac_out]
+
+    def test_check_output(self):
+        self.check_output()
+
+        PassVersionChecker.IsCompatible('conv_affine_channel_fuse_pass')
```
Comment (shangzhizhou) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_affine_channel_fuse_pass.py]: 需要添加断言
self.assertTrue

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_affine_channel_fuse_pass.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
```
Comment (shangzhizhou) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_affine_channel_fuse_pass.py]: 同上

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_affine_channel_fuse_pass.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
```
Comment (shangzhizhou) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_affine_channel_fuse_pass.py]: 同上，以下相同情况请统一修复

**Review by shangzhizhou — DISMISSED** [review]
LGTM

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**